package.json

```diff
@@ -41,6 +42,7 @@
     "dpdm": "^3.14.0",
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.9.1",
+    "esbuild": "^0.16.10",
```

Comment:
can we bump this to 0.20? since we use `"esbuild-loader": "^4.1.0",`, it should be possible [apparently](https://github.com/privatenumber/esbuild-loader/releases).

Reply:
Yeah, I initally also upgraded to the 0.20 version but during the debug session regarding the tests we tried a lot of ways to fix this and this included downgrading to the previously used esbuild version. Now I forget to test whether it is ok to bump esbuild up. 
Thanks for being so nitpicky :D => The 0.20 version seems to be working alright